app/controllers/notification.py

```diff
@@ -68,6 +70,10 @@ async def notify(
             models.Notification(**data.model_dump()),
             auto_commit=True,
         )
+        channels.publish(  # type: ignore
+            {"user_id": str(user.id), "id": str(notification.id), "event": "created"},
```

Comment:
Do we want some kind of ENUM for the different events, so there's a list somewhere for easy reference, and less risks of typos?